version.json

```diff
@@ -3,7 +3,8 @@
 	"version": "2.6",
 	"publicReleaseRefSpec": [
 		"^refs/heads/release$",
-		"^refs/heads/main$"
+		"^refs/heads/main$",
+		"^refs/heads/patch/.*$"
```

Comment:
the patch branches will get 'real' version numbers so we can release out of them.  We also need to setup sync'ing to the internal branch for any patch branch.

Reply:
Just to confirm my understanding of the docs here, this doesn't necessarily change the algorithm used to choose the version number, but just says that builds out of this branch don't get the commit SHA tagged to them?

Reply:
Thats correct